Hmmm.. I've unable to install it into the regular airflow image

```console
❯ docker run -it --rm apache/airflow:2.5.0-python3.9 bash

airflow@7e0bc7525889:/opt/airflow$ uname -a
Linux 7e0bc7525889 5.10.76-linuxkit #1 SMP PREEMPT Mon Nov 8 11:22:26 UTC 2021 aarch64 GNU/Linux

airflow@7e0bc7525889:/opt/airflow$ pip install azure-servicebus
Defaulting to user installation because normal site-packages is not writeable
Collecting azure-servicebus
  Downloading azure_servicebus-7.8.1-py3-none-any.whl (210 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 210.4/210.4 kB 2.3 MB/s eta 0:00:00
Requirement already satisfied: azure-core<2.0.0,>=1.14.0 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (1.26.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (4.4.0)
Requirement already satisfied: six>=1.11.0 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (1.16.0)
Requirement already satisfied: isodate>=0.6.0 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (0.6.1)
Requirement already satisfied: azure-common~=1.1 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (1.1.28)
Requirement already satisfied: msrest<2.0.0,>=0.6.17 in /home/airflow/.local/lib/python3.9/site-packages (from azure-servicebus) (0.7.1)
Collecting uamqp<2.0.0,>=1.5.1
  Downloading uamqp-1.6.3.tar.gz (4.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.4/4.4 MB 3.7 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: requests>=2.18.4 in /home/airflow/.local/lib/python3.9/site-packages (from azure-core<2.0.0,>=1.14.0->azure-servicebus) (2.28.1)
Requirement already satisfied: certifi>=2017.4.17 in /home/airflow/.local/lib/python3.9/site-packages (from msrest<2.0.0,>=0.6.17->azure-servicebus) (2022.9.24)
Requirement already satisfied: requests-oauthlib>=0.5.0 in /home/airflow/.local/lib/python3.9/site-packages (from msrest<2.0.0,>=0.6.17->azure-servicebus) (1.3.1)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/airflow/.local/lib/python3.9/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.14.0->azure-servicebus) (1.26.13)
Requirement already satisfied: charset-normalizer<3,>=2 in /home/airflow/.local/lib/python3.9/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.14.0->azure-servicebus) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in /home/airflow/.local/lib/python3.9/site-packages (from requests>=2.18.4->azure-core<2.0.0,>=1.14.0->azure-servicebus) (3.4)
Requirement already satisfied: oauthlib>=3.0.0 in /home/airflow/.local/lib/python3.9/site-packages (from requests-oauthlib>=0.5.0->msrest<2.0.0,>=0.6.17->azure-servicebus) (3.2.2)
Building wheels for collected packages: uamqp
  Building wheel for uamqp (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for uamqp (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [101 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-aarch64-cpython-39
      creating build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/mgmt_operation.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/message.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/address.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/errors.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/session.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/receiver.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/utils.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/__init__.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/types.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/connection.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/compat.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/sender.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/constants.py -> build/lib.linux-aarch64-cpython-39/uamqp
      copying uamqp/client.py -> build/lib.linux-aarch64-cpython-39/uamqp
      creating build/lib.linux-aarch64-cpython-39/uamqp/authentication
      copying uamqp/authentication/common.py -> build/lib.linux-aarch64-cpython-39/uamqp/authentication
      copying uamqp/authentication/cbs_auth.py -> build/lib.linux-aarch64-cpython-39/uamqp/authentication
      copying uamqp/authentication/__init__.py -> build/lib.linux-aarch64-cpython-39/uamqp/authentication
      copying uamqp/authentication/cbs_auth_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/authentication
      creating build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/session_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/connection_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/receiver_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/utils.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/__init__.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/mgmt_operation_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/client_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      copying uamqp/async_ops/sender_async.py -> build/lib.linux-aarch64-cpython-39/uamqp/async_ops
      running egg_info
      writing uamqp.egg-info/PKG-INFO
      writing dependency_links to uamqp.egg-info/dependency_links.txt
      writing requirements to uamqp.egg-info/requires.txt
      writing top-level names to uamqp.egg-info/top_level.txt
      reading manifest file 'uamqp.egg-info/SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      writing manifest file 'uamqp.egg-info/SOURCES.txt'
      copying uamqp/c_uamqp.c -> build/lib.linux-aarch64-cpython-39/uamqp
      running build_ext
      will build uamqp in build/temp.linux-aarch64-cpython-39/cmake
      Building with generator flags: -G "Unix Makefiles"
      calling cmake /tmp/pip-install-bblny97h/uamqp_6cd81584b2bb407a9ad505cc6952e587/src/vendor/azure-uamqp-c/ -G "Unix Makefiles" -Duse_openssl:bool=ON -Duse_default_uuid:bool=ON  -Duse_builtin_httpapi:bool=ON  -Dskip_samples:bool=ON -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE -DCMAKE_BUILD_TYPE=Release
      CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
      CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
      CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
      -- Configuring incomplete, errors occurred!
      See also "/tmp/pip-install-bblny97h/uamqp_6cd81584b2bb407a9ad505cc6952e587/build/temp.linux-aarch64-cpython-39/cmake/CMakeFiles/CMakeOutput.log".
      Traceback (most recent call last):
        File "/home/airflow/.local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 351, in <module>
          main()
        File "/home/airflow/.local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 333, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/airflow/.local/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 249, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 413, in build_wheel
          return self._build_with_temp_dir(['bdist_wheel'], '.whl',
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 398, in _build_with_temp_dir
          self.run_setup()
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 484, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 260, in <module>
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1208, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/wheel/bdist_wheel.py", line 325, in run
          self.run_command("build")
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1208, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build.py", line 132, in run
          self.run_command(cmd_name)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1208, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-3uhdze9t/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "<string>", line 134, in run
        File "<string>", line 183, in build_cmake
        File "/usr/local/lib/python3.9/subprocess.py", line 373, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command 'cmake /tmp/pip-install-bblny97h/uamqp_6cd81584b2bb407a9ad505cc6952e587/src/vendor/azure-uamqp-c/ -G "Unix Makefiles" -Duse_openssl:bool=ON -Duse_default_uuid:bool=ON  -Duse_builtin_httpapi:bool=ON  -Dskip_samples:bool=ON -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE -DCMAKE_BUILD_TYPE=Release' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for uamqp
Failed to build uamqp
ERROR: Could not build wheels for uamqp, which is required to install pyproject.toml-based projects
```

Maybe better keep requirement without any changes for now, create additional extra where we do not limit platform and later I could investigate and add into the Azure provider documentation about additional requirements for install it on Linux ARM.

WDYT?

--- a/airflow/providers/microsoft/azure/provider.yaml
+++ b/airflow/providers/microsoft/azure/provider.yaml
@@ -63,8 +63,7 @@ dependencies:
   - azure-storage-blob>=12.14.0
   - azure-storage-common>=2.1.0
   - azure-storage-file>=2.1.0
-  # Limited due to https://github.com/Azure/azure-uamqp-python/issues/191
-  - azure-servicebus>=7.6.1; platform_machine != "aarch64"
+  - azure-servicebus>=7.6.1
   - azure-synapse-spark
   - adal>=1.2.7
 

--- a/generated/provider_dependencies.json
+++ b/generated/provider_dependencies.json
@@ -440,7 +440,7 @@
       "azure-mgmt-datafactory>=1.0.0,<2.0",
       "azure-mgmt-datalake-store>=0.5.0",
       "azure-mgmt-resource>=2.2.0",
-      "azure-servicebus>=7.6.1; platform_machine != \"aarch64\"",
+      "azure-servicebus>=7.6.1",
       "azure-storage-blob>=12.14.0",
       "azure-storage-common>=2.1.0",
       "azure-storage-file>=2.1.0",

